kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/device/ThreadsTabUi.java

```diff
@@ -142,4 +142,9 @@ public void onSuccess(ArrayList<GwtGroupedNVPair> result) {
             }
         });
     }
+
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"